src/components/auth/AuthProvider.tsx

```
@@ -3,6 +3,7 @@ import { onAuthStateChanged, User, IdTokenResult } from "@firebase/auth";
 import React, { createContext, useContext, useEffect, useState } from "react";
 import { auth } from "@/lib/firebase/firebaseConfig";
 import Loading from "../Loading";
+import Error from "../Error";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)